docker/android/Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM ubuntu:zesty
+# sys_image can be overridden from the CLI
+ARG sys_image=debian:jessie
```

Comment:
According to the [Docker docs](https://docs.docker.com/engine/reference/builder/#format) this is the right way to do it. Could you please share what `docker --version` and `docker version` you have installed. Also what is your host operating system that you are using ?

Thanks